assets/js/blocks/checkout/inner-blocks/checkout-collection-method/block.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "woocommerce/checkout-collection-method",
+	"version": "1.0.0",
+	"title": "Shipping Method",
```

Comment:
We have a mix of Shipping Options (block defaultTitle), Shipping Method (here), and Collection Method. Can we settle on one, even though it might change in the future?

Reply:
We can handle this in a follow up.